projects/akita-ng-fire/src/lib/auth/auth.service.ts

```diff
@@ -283,12 +280,17 @@ export class FireAuthService<S extends FireAuthState> {
       } else if (this.onSignin) {
         await this.onSignin(cred);
       }
+      /* If this is not a new user, update the store */
+      if (!cred.additionalUserInfo.isNewUser) {
+        profile = await this.createProfile(cred.user);
```

Comment:
createProfile is supposed to create a profile in firebase format or in application format?

If firebase format, then, the this.formatFromFirestore should also be called @line 272
If application format, then createProfile should not be called.

IMHO: If you retrieve a user from firebase with an existing profile, I would NOT expect to have to "createProfile". Create profile should be used to initialize a brand new profile, with default values that make sense.
At least it would have been my understanding.